### PR TITLE
Implement: SNS Publish Batch (bumps aws-sdk => 2.1067.0)

### PR DESCRIPTION
**Pull request by meio-velarde**
Upgrades the SNSQueueClient sendBatchMessage implementation to accommodate AWS SDK's new `publishBatch()` feature for SNS

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -47,7 +47,7 @@
     "peer": {
       "@typescript-eslint/eslint-plugin": ">=4 || <5",
       "@typescript-eslint/parser": ">=4 || <5",
-      "aws-sdk": ">2.900.0 || <3",
+      "aws-sdk": ">2.1067.0 || <3",
```
Comment (fean) [inline package.json:50]: Maybe you could check what the minum supported version is that has all the features you need 🙂

**Review by fean — CHANGES_REQUESTED** [review]

File: services/queue/SNSQueueClient.ts [hunk 2]
```diff
@@ -67,13 +76,28 @@ class SNSQueueClient implements QueueClient {
...
+          request.options?.attributes,
+        )
+      : {}
 
-    await Promise.all(batchSendMessagePromises)
+    const batchSendMessageEntries = request.messages.map((message) => ({
+      Id: uuidV1(),
+      Message: this.serializer.serialize(message, 'string'),
+      MessageGroupId: request.options?.messageGroupId,
+      MessageDeduplicationId: request.options?.messageGroupId,
```
Comment (lueue) [inline services/queue/SNSQueueClient.ts:91]: a groupId and deduplicationId should be something different I think. They have different meanings

Comment (fean) [reply services/queue/SNSQueueClient.ts:91]: Good catch!

**Review by fean — APPROVED** [review]
Looks good man! 💪

**Pull request merged**